options/simple_options.hpp

```diff
@@ -47,6 +47,7 @@ namespace bzn::option_names
     const std::string WS_IDLE_TIMEOUT = "ws_idle_timeout";
     const std::string PEER_VALIDATION_ENABLED = "peer_validation_enabled";
     const std::string SIGNED_KEY = "signed_key";
+    const std::string BLUZELLE_PUBLIC_KEY = "bluzelle_public_key";
```

Comment:
This should be something like owner_public_key.

Reply:
Changed to owner_public_key and propagated the changes.